### ISSUE TITLE
tl.command("build.updatebuildnumber") yields an error on latest Windows Agent

### ISSUE THREAD

**Issue by jessehouwing** [Microsoft/vsts-agent#1299]
Reported here: https://github.com/jessehouwing/vsts-variable-tasks/issues/28

**Comment by TingluoHuang** [comment]
@jessehouwing the agent code for handle ##vso[build.updatebuildnumber] haven't change for long long time.
base on the conversation on https://github.com/jessehouwing/vsts-variable-tasks/issues/28, it sounds you had a bug in your PS implementation at one point, when customer use your task on windows, that is causing the failure.

**Comment by jessehouwing** [comment]
The cause was that the original tasks were using `##vso[build.updatebuildnumber;]` note the `;` and that doesn't seem to be supported (anymore?), these tasks shipped with the 1.80 agent originally. It does indeed seem that the tasks did not get updated on their end, after updating the tasks again, the new version got downloaded.

I think I originally included the `;` based on the docs here: https://github.com/Microsoft/vsts-tasks/blob/master/docs/authoring/commands.md which show a semicolon at the end of the two example commands. 

It does seem that the last 3 commands indeed do not end with `;`. properly confusing.

**Comment by TingluoHuang** [comment]
Sure, we can improve that.

But i do give a sample for `build.updatebuildnumber` which has the correct syntax.
 
https://github.com/Microsoft/vsts-tasks/blob/master/docs/authoring/commands.md#build-logging-commands
```
Example: 
##vso[build.updatebuildnumber]my-new-build-number
```
